crates/cli/src/lib.rs

```diff
@@ -138,6 +138,7 @@ fn run_compiler_with(args: Args, f: impl FnOnce(&Compiler) -> Result + Send) ->
     sess.stop_after = args.stop_after;
     sess.dump = args.unstable.dump.clone();
     sess.ast_stats = args.unstable.ast_stats;
+    sess.storage_sz_ub = args.unstable.storage_sz_ub;
```

Comment:
This region wouldn't run `#[cfg(test)]`, what can I do ?

Reply:
It can't and shouldn't be cfg(test)